test/selenium-lib.js

```diff
@@ -21,7 +21,8 @@ function buildDriver() {
     return sharedDriver;
   }
   // Firefox options.
-  // http://selenium.googlecode.com/git/docs/api/javascript/module_selenium-webdriver_firefox.html
+  // http://selenium.googlecode.com/git/docs/api/javascript/
+  //  module_selenium-webdriver_firefox.html
```

Comment:
Fixed. 

Not sure there could be a rule to check for this (stuff like URLs on multiple lines) in comments.